ui/analyse/src/study/relay/relayTourView.ts

```diff
@@ -319,11 +321,11 @@ const games = (ctx: RelayViewContext) => [
     ? h(
         'div.relay-tour__note',
         h('div', [
-          h('div', 'No boards yet. These will appear once games are uploaded.'),
+          h('div', i18n.broadcast.noBoardsYet),
           ctx.study.members.myMember() &&
             h('small', [
-              'Boards can be loaded with a source or via the ',
-              h('a', { attrs: { href: '/broadcast/app' } }, 'Broadcaster App'),
+              i18n.broadcast.boardsCanLoaded,
+              h('a', { attrs: { href: '/broadcast/app' } }, ' Broadcaster App'),
```

Comment:
try
```typescript
  h(
    'small',
    i18n.broadcast.boardsCanLoaded.asArray(
      h('a', { attrs: { href: '/broadcaster/app' } }, 'Broadcaster App')
    )
  ),

Reply:
i didn't look at the boardsCanLoaded string but it seems like you want to insert a link inside of it.

Reply:
oh, i see you're just talking about the space thing. I believe whitespace has always been trimmed from translations, so you may have needed that space regardless.